changelog/preview-in.dd

```diff
@@ -0,0 +1,13 @@
+`-preview=in` now makes the `in` storage class mean `scope const`.
```

Comment:
That might just be me, but that phrasing could be interpreted like a change to an existing functionality, instead of the addition of a new preview switch. I'd find something along the lines of:
```suggestion
Added `-preview=in` to make the `in` storage class mean `scope const` again.
```
less ambiguous (the "again" can be lost if you want)